buildtools/cli.go

```diff
@@ -934,9 +934,18 @@ func NpmPublishCmd(c *cli.Context) (err error) {
 	if npmCmd.GetXrayScan() {
 		commandsUtils.ConditionalUploadScanFunc = scan.ConditionalUploadDefaultScanFunc
 	}
-	printDeploymentView, detailedSummary := log.IsStdErrTerminal(), npmCmd.IsDetailedSummary()
-	if !detailedSummary {
-		npmCmd.SetDetailedSummary(printDeploymentView)
+
+	var printDeploymentView, detailedSummary bool
+
+	// we do not support deployment view and detailed summary when using npmrc for publishing since
+	// TransferDetails is not available to us
```

Comment:
```suggestion
	// Deployment view and Detailed summary is not supported when using npmrc for publishing since transfer details are not available.
```